src/qilib/utils/type_aliases.py

```diff
@@ -1,6 +1,19 @@
-from typing import Union, Sequence, List
+from typing import Any, Dict, List, Sequence, Union
+import numpy.typing as npt
 
 PJSValueTypes = Union[None, bool, int, float, complex, str, bytes]
 PJSContainers = Sequence[PJSValueTypes]
 PJSValues = Union[PJSContainers, PJSValueTypes]
 TagType = List[str]
+EncodedNumpyArray = Dict[str, Union[str, Dict[str, Union[str, List[Any], bytes]]]]
+# type alias for numpy.ndarray
+numpy_ndarray_type = npt.NDArray[Any]
```

Comment:
What makes this type definition different from the ones above, giving it a different naming convention?

Reply:
Historical reasons of course. This type definition was in a 'normal' python file, where other types were defined in type aliases. I only moved it to type_aliases without renaming the type. It can be renamed of course to be more consistent in naming types I agree. Will do a search and replace